doc/classes/Object.xml

```diff
@@ -148,23 +178,76 @@
+				[gdscript]
+				var button = Button.new()
+				button.pressed.connect(_on_Button_pressed) # is the same as:
+				button.connect("pressed", self, "_on_Button_pressed")
+
+				var line_edit = LineEdit.new()
+				line_edit.text_entered.connect(_on_LineEdit_text_entered) # is the same as:
+				line_edit.connect("text_entered", self, "_on_LineEdit_text_entered")
+
+				# Assuming you have a Player class that defines the `hit` signal.
+				var player = Player.new()
+				var weapon_type = "sword"
+				var damage = 100
+				player.hit.connect(_on_Player_hit.bind(weapon_type, damage)) # is the same as:
+				player.connect("hit", self, "_on_Player_hit", [weapon_type, damage])
+				[/gdscript]
+				[csharp]
+				var button = new Button();
+				button.Pressed += OnButtonPressed;
+				button.Connect("pressed", this, nameof(OnButtonPressed));
+
+				var lineEdit = new LineEdit();
+				lineEdit.TextEntered += OnLineEditTextEntered;
+				lineEdit.Connect("text_entered", this, nameof(OnLineEditTextEntered))
+
+				var player = new Player(); // assuming you have a Player class that defines the hit signal.
+				string weaponType = "sword";
+				int damage = 100;
+				// Signals as events (`player.hit += OnPlayerHit;`) do not support argument binding. You can only use:
+				player.Connect("hit", this, nameof(OnPlayerHit), new Godot.Collections.Array {weaponType, Damage });
+				[/csharp]
+				[/codeblocks]
 				An example of the relationship between [code]binds[/code] passed to [method connect] and parameters used when calling [method emit_signal]:
-				[codeblock]
-				connect("hit", self, "_on_Player_hit", [ weapon_type, damage ]) # weapon_type and damage are passed last
-				emit_signal("hit", "Dark lord", 5) # "Dark lord" and 5 are passed first
+				[codeblocks]
+				[gdscript]
+				func _ready():
+				    var player = Player.new() # assuming you have a Player class that defines the hit signal.
+				    var weapon_type = "sword"
+				    var damage = 100
+				    player.hit.connect(_on_Player_hit.bind(weapon_type, damage)) # is the same as:
+				    player.connect("hit", self, "_on_Player_hit", [ weapon_type, damage ])
+
+				    player.emit_signal("hit", "Dark lord", 5) # "Dark lord" and 5 are passed first
+
+
 				func _on_Player_hit(hit_by, level, weapon_type, damage):
 				    print("Hit by %s (lvl %d) with weapon %s for %d damage" % [hit_by, level, weapon_type, damage])
-				[/codeblock]
+				[/gdscript]
+				[csharp]
+				public override void _Ready()
+				{
+				    var player = new Player(); // assuming you have a Player class that defines the hit signal.
+				    var weaponType = "sword";
+				    int damage = 100;
+				    player.Hit.Connect(new Callable(this, new Callable(this, nameof(OnPlayerHit)).Bind(weaponType, damage)); // is the same as:
+				    player.Connect("hit", this, nameof(OnPlayerHit), new Godot.Collections.Array{ weaponType, damage });
+
+				    Player.EmitSignal("hit", "Dark lord", 5); // "Dark lord" and 5 are passed first
```

Comment:
Type, `Player` should be `player` (not capitalized).

Reply:
fixed